orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/entity/OrchSessionItem.java

```diff
@@ -102,4 +104,18 @@ public OrchSessionItem withAccountState(AccountState accountState) {
         this.isNewAccount = accountState;
         return this;
     }
+
+    @DynamoDbAttribute(ATTRIBUTE_INTERNAL_COMMON_SUBJECT_ID)
+    public String getInternalCommonSubjectId() {
+        return internalCommonSubjectIdentifier;
+    }
+
+    public void setInternalCommonSubjectId(String internalCommonSubjectIdentifier) {
```

Comment:
This one too - rename to `internalCommonSubjectId`

Reply:
fixed